### ISSUE TITLE
Decide what to do about the RxJS documentation

### ISSUE THREAD

**Issue by DavidMGross** [ReactiveX/reactivex.github.io#247]
the RxJS team have pursued their own documentation path and are doing some great work (http://reactivex.io/rxjs/). Does it still make sense to try to maintain our parallel documentation here? Currently we have RxJS accordion sections for all of the operator pages, for instance, but those aren't being maintained by the RxJS team. Maybe they should just link to the documentation at http://reactivex.io/rxjs/.

**Comment by staltz** [comment]
I agree. I find the accordion to be quite horrible to be honest. I think on each operator doc page we should just link to the corresponding RxJS operator doc page. I also think we should do the same for RxJava, since when I'm using RxJava I always go to the official JavaDoc page instead of reactivex.io.

Opinions? @blesh

**Comment by staltz** [comment]
By the way, I was in the middle of updating the accordion to match RxJS v5 operators (see https://github.com/staltz/reactivex.github.io/commits/rxjs5) but I aborted that plan to work solely on reactivex.io/rxjs, because maintaining the accordion would be always duplicate work, framed in a worse format, and susceptible to becoming easily outdated.

**Comment by DavidMGross** [comment]
The RxJava / RxGroovy accordion sections for the operators already include
links to the javadocs.

One thing that's nice about the reactivex.io site compared to the javadocs
is that it groups operators with similar functions but dissimilar names, so
that for example on the "flatMap" page you also see the documentation for
concatMap and switchMap. This way if you're investigating which operator to
use for a particular task, you'll see all the related operators in one
place rather than having to hunt around the huge Observable javadoc page
for them.

On Fri, May 20, 2016 at 3:58 AM, André Staltz <notifications@github.com>
wrote:

> By the way, I was in the middle of updating the accordion to match RxJS v5
> operators (see https://github.com/staltz/reactivex.github.io/commits/rxjs5)
> but I aborted that plan to work solely on reactivex.io/rxjs, because
> maintaining the accordion would be always duplicate work, framed in a worse
> format, and susceptible to becoming easily outdated.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/reactivex.github.io/issues/247#issuecomment-220577111>
>



-- 
David M. Gross
PLP Consulting

**Comment by blesh** [comment]
> I think on each operator doc page we should just link to the corresponding RxJS operator doc page.

This sounds fine to me.

I guess I'm not sure what the conflicting proposals here are.

**Comment by EFFKAY2** [comment]
As a newcomer to the rx world, I happen to think that the documentation here is easier to understand than the docs at rxjs. For a quick overview of how an operator works I come here, and if further detail is needed I go there. The diagrams and explenations are much better here.
